charts/prefect-server/templates/deployment.yaml

```diff
@@ -65,6 +65,9 @@ spec:
             - prefect
             - server
             - start
+            {{- if .Values.server.loopServices.enabled }}
```

Comment:
I think we need a better name than `enabled`; reading "if services enabled then no services" is easy to get confused

Reply:
agree. thinking....